certdocs/0-DDLN-100-ORD.md

```diff
@@ -62,7 +62,34 @@ Reqtraq's goal is to ensure bidirectional traceability throughout all levels of
 
 Functional requirements identify what is necessary to obtain the desired performance of the system under the listed operational modes and conditions. This category includes and is developed through a combination of customer, operational, performance, physical and installation, maintainability, security, and interface restrictions, wishes, and requirements. Functional requirements are defined and written according the section 5.3 of ARP 4754A / ED-79A.
 
-#### REQ-0-DDLN-SYS-001. Bidirectional tracing.
+#### REQ-RQTQ-SYS-7 Requirement Document Types
+
+The RMT SHALL allow tracking of the following requirements document types:
+
+- Overall Requirements Document, which contains <abbr title="system-level requirements">SYS</abbr>
```

Comment:
yes this is the mapping i was referring to that feels confusing to me / another thing to memorize